sqlglot/parser.py

```diff
@@ -5916,6 +5960,41 @@ def _parse_alter_table_rename(self) -> t.Optional[exp.RenameTable | exp.RenameCo
         self._match_text_seq("TO")
         return self.expression(exp.RenameTable, this=self._parse_table(schema=True))
 
+    def _parse_alter_table_set(self) -> exp.AlterSet:
+        alter_set = self.expression(exp.AlterSet)
+
+        if self._match(TokenType.L_PAREN, advance=False) or self._match_text_seq(
+            "TABLE", "PROPERTIES"
+        ):
+            alter_set.set("expressions", self._parse_wrapped_csv(self._parse_conjunction))
+        elif self._match_text_seq("FILESTREAM_ON", advance=False):
+            alter_set.set("expressions", [self._parse_conjunction()])
+        elif self._match_texts(("LOGGED", "UNLOGGED")):
+            alter_set.set("option", exp.var(self._prev.text.upper()))
+        elif self._match_text_seq("WITHOUT") and self._match_texts(("CLUSTER", "OIDS")):
+            alter_set.set("option", exp.var(f"WITHOUT {self._prev.text.upper()}"))
```

Comment:
I'd consider using `_parse_var_from_options` for these two and move it to the `else` branch. That way, it'll also make it easier to extend without creating clutter (ie. more branches) in the future if needed.

Reply:
That was my initial implementation but it resulted to a very small dictionary as it's only Postgres that supports these few static options; Thus, I thought that a stray options dict would create more clutter than these couple branches for that matter. Wdyt?

Reply:
Ok, let's leave it as is.